### PR TITLE
create basic aid request model

### PR DESCRIPTION
**Pull request by bluquar**
Continuing to follow demo at https://www.digitalocean.com/community/tutorials/build-a-to-do-application-using-django-and-react

Screen Shot 2021-11-20 at 3 49 38 AM

### CODE REVIEW

File: backend/community_aid_backend/models.py [hunk 1]
```diff
@@ -1,3 +1,12 @@
 from django.db import models
 
 # Create your models here.
+
+class AidRequest(models.Model):
+    what_is_needed = models.CharField(max_length=120)
```
Comment (lowellbander) [inline backend/community_aid_backend/models.py:6]: #26

File: backend/community_aid_backend/models.py [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+    completed = models.BooleanField(default=False)
+    zip_code = models.PositiveIntegerField()
```
Comment (lowellbander) [inline backend/community_aid_backend/models.py:8]: #27

**Review by lowellbander — APPROVED** [review]
fab 👏 u 👏 lous

left some inline comments linking to new GH issues

at this point the repo name is a misnomer, yeah? Because backend is in here too, so rep is community-aid-app or sth?

**Comment by bluquar** [comment]
yeah the repo name is a misnomer. do u think it makes sense to have them in separate repos? I think it might because at this point, the frontend is going to be served from a node server, whereas the "backend" (for async calls) will be served by a django server, so we could have them running in separate containers/processes.

**Comment by lowellbander** [comment]
I honestly don't know if separate repos is a prereq for separate containers/servers, so the best i can say is that separate repos "seems" like a good idea; created #32

**Pull request merged**